index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body class="body">
+    <div class="first-part">
+        <header class="topnav">
+          <a href="index.html"><img class="welcome" src="img/Welcome.png"></a>
+          <nav>
+            <a href="#"><img class="Menu" src="img/Icon - Menu.png"></a>
+          </nav>       
+        </header>
+        <section class="aboutMe">
+          <h1>Hey there. I'm Nilton<span> I'm a Full Stack web developer</span></h1>
+          <p> I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [OPTIONAL] Kindly avoid lines of code longer than 120 characters. Your code will be easier to read and follow the best practices.